src/main/kotlin/edu/byu/mealplanningassistant/services/MealPlanGenerator.kt

```diff
@@ -0,0 +1,22 @@
+package edu.byu.mealplanningassistant.services
+
+import edu.byu.mealplanningassistant.models.GetRandomizedRecipesRequest
+import edu.byu.mealplanningassistant.models.GetRandomizedRecipesResponse
+
+class MealPlanGenerator {
+    fun getRandomizedRecipes(getRandomizedRecipesRequest: GetRandomizedRecipesRequest) : GetRandomizedRecipesResponse {
+        TODO()
+        /**
+         * queryRecordsInRandomOrder().forEach {
+         *      if it.matches(getRandomizedRecipeRequests.requests.any()) {
+         *          assign(it to the first recipe request that matches that has no recipes assigned)
+         *      }
+         *      if all recipe requests are filled {
```

Comment:
What does all recipe requests being filled look like? Are we looking for a certain number of recipes? Sorry if this is a dumb question

Reply:
That's a great question and it's something we still have to decide on. `GetRecipesRequest`s are made of `GetRecipeRequest`s (the naming might need to change or we might have to forgo the singular one and just make singular ones use the plural request with a single meal requested). When all of those have a recipe associated with them, then we can format that in a `GetRecipesResponse` and return it to the user.